pkg/registry/registrytest/etcd.go

```diff
@@ -42,11 +42,11 @@ func NewEtcdStorageForResource(t *testing.T, resource schema.GroupResource) (*st
 	completedConfig.APIResourceConfig = serverstorage.NewResourceConfig()
 	factory, err := completedConfig.New()
 	if err != nil {
-		t.Fatal(err)
+		t.Fatalf("Error while making storage factory: %v", err)
 	}
 	resourceConfig, err := factory.NewConfig(resource)
 	if err != nil {
-		t.Fatal(err)
+		t.Fatalf("Error while finding storage destination: %w", err)
```

Comment:
vet is catching this:
```suggestion
		t.Fatalf("Error while finding storage destination: %v", err)
```

Reply:
Done.

Reply:
Sorry, I had an adventure with rebasing and then forgot to push this change back down to this PR.  This is done now.